delphi/translators/for2py/scripts/genPGM.py

```diff
@@ -759,6 +764,7 @@ def genPgm(node, state, fnNames):
     # Name: ('id', 'ctx')
     elif isinstance(node, ast.Name):
         lastDef = getLastDef(node.id, state.lastDefs, state.lastDefDefault)
+        print(state.lastDefs)
```

Comment:
@pratikbhd Is this a debugging print statement, or is this part of the program logic?  If the former, can you delete it?

Reply:
Yes, this is a debugging print statement. I seemed to have missed deleting this one. Just deleted.

Reply:
Thanks. :)